### DISCUSSION TITLE
ephem.4.32       probably  obsolete  but still useful

### DISCUSSION THREAD

**Discussion by oystercatcher** [XEphem/XEphem#69] [General]
https://clearskyinstitute.com/ephem/

I actually prefer  to run  the console based program.   I had some minor problems
with  the  previous release which used  cmake   but  4.32  builds  on  fedora37

I remember reading  years ago  how to setup the symbolic links  with  /usr/lib64  to compile  Xephem.  

Well ephem.4.32  probably the accuracy is not the greatest but  I can see  alt  az  for  all the major planets
and  their rise and set times.       

Also  my goto  for  kids  is stellarium  but the web interface is
limited   and   standalone won't run  if  opengl  is not supported.        

I am using  the  replacement  for pyephem  --  skyfield

**Comment by ghost** [comment]
Thanks for the note. Always nice to hear of someone still using ephem; minimal still has its place.

**Comment by zvezdochiot** [comment]
Hi @oystercatcher .

"Reposted" to git: https://github.com/Geo-Linux-Calculations/ephem

**Comment by ghost** [comment]
@zvezdochiot wow, many thanks for posting.

**Comment by brandon-rhodes** [comment]
I have fond memories of sitting with my grandfather near the end of his life, in the early 1990s, and learning what we could observe on a given evening by running ephem. The green letters would fill the screen of his Unix PC. The table of positions would come up rather quickly. But the `Rise Time` `Set Time` table would draw more slowly—you could see each row fill separately, with one pause after the object name, then another after the rising circumstances, then another after the transit circumstances. Then all at once the setting circumstances, the hours-up, and the next row's object name would print in a single spurt, and then you'd be waiting again for the next row.

It provided a bit of suspense if you were waiting for, say, the position of a comet, which you had assigned to `X` or `Y` at the very bottom of the screen.

**Comment by oystercatcher** [comment]
I have  a  raspberrypi4  with  bullseye   
sudo  apt install   libncurses-dev

make  compiles   ephem  no problem and it is running  on  32 bit  rpi
